string object_name
geometry_msgs/Pose init_pose
string init_frame
int8 track
int8 once
---
int32 responseId

